folly/container/SparseByteSet.h

```diff
@@ -77,6 +78,15 @@ class SparseByteSet {
     return sparse_[i] < size_ && dense_[sparse_[i]] == i;
   }
 
+  /***
+   *  clear()
+   *
+   *  O(1), non-amortized.
+   */
+  inline bool clear() {
+    size_ = 0;
```

Comment:
While this will make it always return false for all values, it would fail if you try to insert anything into it again. If clearing is needed, then this would need to use versioning for the sparse marker rather than just storing the `size_`.

Reply:
Fix return type. 
But I don't understand what are you talking about. Can you explain a little bit more? Why my solution should not to work?